src/main/java/com/gameshop/ecommerce/user/service/MyAccountService.java

```diff
@@ -0,0 +1,115 @@
+package com.gameshop.ecommerce.user.service;
+
+import com.gameshop.ecommerce.address.Address;
+import com.gameshop.ecommerce.address.dto.AddressDto;
+import com.gameshop.ecommerce.security.EncryptionService;
+import com.gameshop.ecommerce.user.dao.LocalUserDAO;
+import com.gameshop.ecommerce.user.model.LocalUser;
+import com.gameshop.ecommerce.user.mapper.LocalUserMapper;
+import com.gameshop.ecommerce.user.model.LocalUserDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+@Slf4j
+public class MyAccountService {
+
+    private final LocalUserDAO localUserDAO;
+
+    private final LocalUserMapper localUserMapper;
+
+    private final EncryptionService encryptionService;
+
+    @Autowired
+    public MyAccountService(LocalUserDAO localUserDAO, LocalUserMapper localUserMapper, EncryptionService encryptionService) {
+        this.localUserDAO = localUserDAO;
+        this.localUserMapper = localUserMapper;
+        this.encryptionService = encryptionService;
+    }
+
+    public LocalUserDto getInfo(UUID userId){
+        LocalUser user = localUserDAO.findById(userId)
+                .orElseThrow(() -> new RuntimeException("User not found"));
+
+        return userBuilder(user);
+    }
+
+    public void deleteInfo(UUID userId){
+        localUserDAO.deleteById(userId);
+    }
+
+    public LocalUserDto updateInfo(UUID userId, LocalUserDto userDto){
```

Comment:
I think it would be better to rewrite this method and make it more concise and readable